pkg/webui/components/checkbox/group/index.js

```diff
@@ -22,9 +22,25 @@ import style from './group.styl'
 export const CheckboxGroupContext = React.createContext()
 
 const CheckboxGroup = props => {
-  const { children, className, horizontal, onChange, disabled, onFocus, onBlur, ...rest } = props
-  const hasValue = 'value' in rest
-  const [value, setValue] = React.useState(hasValue ? rest.value : rest.initialValue || {})
+  const {
+    children,
+    className,
+    horizontal,
+    onChange,
+    disabled,
+    onFocus,
+    onBlur,
+    value: valueProp,
+    ...rest
+  } = props
+  const hasValue = Boolean(valueProp)
+  const [value, setValue] = React.useState(hasValue ? valueProp : rest.initialValue || {})
+
+  React.useEffect(() => {
+    if (valueProp) {
+      setValue(valueProp || {})
+    }
```

Comment:
Looking at the whole implementation (also from before the refactor) it's kind of unclear to me why we use state here anyway. I believe this is some relict of our storybook stories, where we wanted form components to work without any higher-up controller (like formik). Seems like we're jumping quite some hoops here to make it all work together.

I'm fine with quickfixing it like this though for now.

Reply:
I will add this to the refactoring list so we can fix properly in the future.